scripts/scil_bundle_diameter.py

```diff
@@ -201,6 +210,12 @@ def fit_circle_in_space(positions, directions, dist_w=None):
     return center, radius, error
 
 
+def snapshot(scene, win_dims, output_filename):
+    snapshot = next(snapshot_scenes([scene], win_dims))
```

Comment:
These three lines seem to come back often. Could a method 'snapshot' be created somewhere in io.viz or viz.something?

Reply:
These lines appear in scripts that use a legacy way of slicing and specifying rendering options. It's not the way snapshot should be done, those scripts should use `snapshot_scenes` directly, so I won't port it inside the module. Upgrading those scripts should be done in another PR, there is a lot of work to do in them. I can add a TODO or just a simple comment where it happens, would it be fine with you ?

Reply:
Yes